# No.1
# f = open("black.py")
# s = f.read()
# print(s)
# f.close()

# No.2
# f = open("black.py", "r")
# s = f.read()
# print(s)
# f.close()

# No.3
# f = open("black.py", "w")
# s = f.write()
# f.close()

# 评测机
# f = open("black.in")
# s = f.read()
# print(s)
# f.close()
#
# f = open("black.out", "w")
# f.write(s * 3)
# f.close()

# 异常捕捉
# try:
#     < statements1 > 运行想要试错的代码
#
# except < name1 >: 捕获异常name1
#     如果在try部分引发了'name1'异常，运行此语句
#     < statement2 >
#
# else: 无异常
#     < statement3 > 无异常运行的代码
#
# finally: 不管是否发生异常，保证执行
#     < statement4 > 发布发生异常都会运行的代码

